security/src/main/java/gohigher/jwt/support/CookieProvider.java

```diff
@@ -0,0 +1,33 @@
+package gohigher.jwt.support;
+
+import java.time.Duration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.server.Cookie.SameSite;
+import org.springframework.http.ResponseCookie;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CookieProvider {
+
+	private static final String REFRESH_TOKEN = "refreshToken";
+	private final long refreshTokenExpireLength;
```

Comment:
```suggestion
	private static final String REFRESH_TOKEN = "refreshToken";
	
	private final long refreshTokenExpireLength;
```

가독성 및 용도의 구분을 위해 띄워주면 어떨까요? 그리고 상수 이름으로 REFRESH_TOKEN 은 리프레시 토큰 자체를 의미하는 느낌이 더 큰것 같은데 KEY_OF_REFRESH_TOKEN 이나 REFRESH_TOKEN_KEY 등의 이름은 어떠신가요??